de.j0chn.nextcloud_password_client.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
You should maintain it at https://gitlab.com/j0chn/nextcloud_password_client/. (Also the icon file.)

Reply:
I created a desktop file for snap, which is included to my repo. I will use that instead